enos/templates/Vagrantfile.j2

```diff
@@ -6,28 +6,33 @@
 $script = <<SCRIPT
 cp -r /home/vagrant/.ssh /root/.
 apt-get update
-apt-get install -y apt-transport-https python-setuptools
-easy_install pip && ln -s /usr/local/bin /usr/bin/pip || true
+apt-get install -y apt-transport-https python-setuptools ca-certificates
+easy_install pip && ln -s /usr/local/bin/pip /usr/bin/pip || true
 SCRIPT
 
 Vagrant.configure(2) do |config|
   config.hostmanager.enabled = true
   config.hostmanager.manage_host = true
   config.hostmanager.manage_guest = true
   config.hostmanager.ignore_private_ip = false
-  config.vm.box = "debian/contrib-jessie64"
+  config.vm.box = "debian/jessie64"
   config.vm.provision "shell", inline: $script
 
-	{% for machine in machines %}
-	config.vm.define "{{ machine.name }}" do |my|
+  {% for machine in machines %}
+  config.vm.define "{{ machine.name }}" do |my|
     my.vm.hostname = "{{ machine.name }}"
     my.vm.network :private_network, ip: "{{ machine.ip1 }}"
     my.vm.network :private_network, ip: "{{ machine.ip2 }}"
     my.vm.network :private_network, ip: "{{ machine.ip3 }}"
     my.vm.provider :virtualbox do |vb|
       vb.cpus={{ machine.cpu }}
-			vb.memory={{ machine.mem }}
-		end
+      vb.memory={{ machine.mem }}
+    end
+    my.vm.provider :libvirt do |lv|
```

Comment:
Hey, I make a PR (see #87) that lets you specify such a thing into the reservation.yml. One could do something like that : 
```yaml
provider: 
  type: "vagrant"
  virt: "libvirt"
```
and then get an access to the `virt` value through the `config` object.